src/ResourceManager/GuestConfiguration/Commands.GuestConfiguration/Common/ParameterHelpMessages.cs

```diff
@@ -0,0 +1,30 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.GuestConfiguration.Common
+{
+    /// <summary>
+    /// Parameter help messages
+    /// </summary>
+    public static class ParameterHelpMessages
+    {
+        public const string ResourceGroupName = "Resource group name.";
+        public const string VMName = "VM name.";
+        public const string InitiativeName = "Name of a policy where definition type is Initiative and category is Guest Configuration";
```

Comment:
If these messages are public facing, they should go into the resource strings

Reply:
I tried to use resx strings for these. But it does not work. 
![parameterhelpmessages](https://user-images.githubusercontent.com/25780318/46105511-05fece00-c18b-11e8-9546-ed4a8f034fdc.JPG)